xyz.ayrsd.AudioStellar.yml

```diff
@@ -0,0 +1,319 @@
+app-id: xyz.ayrsd.AudioStellar
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: audiostellar
+finish-args:
+  # We need MIDI, Audio, USB, DRI, and realtime performances
+  - --device=all
+  
+  # We need X11, PulseAudio, JACK and Pipewire support
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0  
+  
+  # We are using the OSC protocol
+  - --share=network
+
+  # Give access to the host filesystem
+  # Many users have files
+  # on removable media and various locations.
+  - --filesystem=host
```

Comment:
then you can restrict to where the removable media are mounted, or make sure to support xdg-document portal. This is a very broad and unecessary permission of a sound editor.

Reply:
thanks @hfiguiere, maybe using "home" is enough?, but how can i make sure removable media will be supported ?